### PR TITLE
feat(code): add missing flags

### PR DESCRIPTION
**Pull request by mrienstra**
Motivated by missing `-` option.

See also:

https://github.com/microsoft/vscode/blob/main/src/vs/platform/environment/node/argv.ts
Complete.

https://github.com/microsoft/vscode/blob/main/cli/src/commands/args.rs
Missing `profile`, maybe more.

https://github.com/microsoft/vscode/blob/main/resources/completions/zsh/_code
Missing `profile`, maybe more.

_(I'll open a PR on `microsoft/vscode` about those later)_

```
>code -v
1.71.2
```

### CODE REVIEW

**Comment by mrienstra** [comment]
Whoops, missed the "conventional commits" requirement, my bad. Should I rebase & force push?

**Comment by mrienstra** [comment]
I tested all new and changed options using `npm run dev`, they all looked fine.

**Comment by fedeci** [comment]
@mrienstra do not worry about conventional commits when you work on your PR. We squash the commits so that part of history gets "lost" (it gets lost from the git history only. It remains in the github PRs history). 
The only thing that should follow semantic commits is the PR title, cause that's the commit message that we use when merging the PR.

File: src/code.ts [hunk 1]
```diff
@@ -1,4 +1,8 @@
 const commonOptions: Fig.Option[] = [
+  {
+    name: ["-"],
```
Comment (fedeci) [inline src/code.ts:3]:
```suggestion
    name: "-",
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks!

**Review by fedeci — APPROVED** [review]
Thanks again!

**Comment by mrienstra** [comment]
Thanks for your help!

**Pull request merged**